extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -219,6 +232,12 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.analyticsGroup_ =
         this.instrumentation_.createAnalyticsGroup(this.element);
 
+    if (this.config_['transport'] && this.config_['transport']['iframe']) {
+      this.transport_ = new Transport(this.element.getAttribute('type'));
+      this.transport_.processCrossDomainIframe(this.getAmpDoc().win,
```

Comment:
to separate concern, should this be moved into Transport, and this method then can be private.